nab-al-tools.code-workspace

```diff
@@ -57,7 +57,7 @@
       "dbaeumer.vscode-eslint",
       "esbenp.prettier-vscode",
       "davidanson.vscode-markdownlint",
-      "eamodio.tsl-problem-matcher"
+      "amodio.tsl-problem-matcher"
```

Comment:
@jwikman  So this is a wierd change but from what I could deduct from my little investigation (nothing found in any changelog or commit history). The marketplace publisher `eamodio` leads to what seems to be Axsoft which publishes GitLens. The publisher `amodio` leads to Eric Amodios personal publisher page. Eric seems to work with Axsoft. So it kinda checks out but it would be nice to se some kind of log for this. Altough I'm not quite sure how long this has been wrong, maybe always?

Reply:
https://marketplace.visualstudio.com/publishers/amodio
https://marketplace.visualstudio.com/publishers/eamodio